acceptance-tests/aws/postgres_validator/index.js

```diff
@@ -0,0 +1,36 @@
+const restify = require('restify');
+const vcapServices = require('vcap_services');
+const testPostgres = require('./test')
+
+function runServer(content) {
+    const server = restify.createServer();
+    server.get('/', (_, res, next) => {
+        res.send(content)
+        next()
+    });
+
+    server.listen(process.env.PORT || 8080, function () {
+        console.log('%s listening at %s', server.name, server.url);
+    });
+}
+
+async function runTest(credentials, testFunc) {
+    try {
+        await testFunc(credentials, runServer)
+    } catch (e) {
+        console.error(e)
+    }
+}
+
+let credentials = vcapServices.findCredentials({ instance: { tags: 'postgres' } });
+
+console.log("postgres credentials", credentials)
+if (Object.keys(credentials).length > 0) {
+    runTest(credentials, testPostgres).then(() => {
+        console.log('Success')
+    }).catch((err) => {
```

Comment:
Given the function is `async` it might be a little more consistent to use `await` here, although I do remember that there were some issues with `async/await` and perhaps this is one of those situations where it doesn't work.

Reply:
Definitely. This was a copy paste of a more complicated app and just simplified some bits. I'll polish this a bit more so it's not confusing.

Reply:
In fact, in order to be able to do an await I have to be inside an async function, which is not the case. I simplified the code though so there's less noise.